src/main/resources/fxml/WorkspaceView.fxml

```diff
@@ -6,8 +6,8 @@
 <?import com.winner_is_kungen.tda367.controller.BlueprintController?>
 <fx:root type="TabPane" tabClosingPolicy="UNAVAILABLE" xmlns:fx="http://javafx.com/fxml/1"
 		 xmlns="http://javafx.com/javafx/11.0.1">
-	<tabs >
-		<Tab text="new file" >
+	<tabs>
+		<Tab text="">
```

Comment:
Would it be better to just remove the "default" tab and have the `BlueprintController` be initiated from code?

Reply:
Cool, it was just a thought but if you agree 👍